lib/ttnt/tasks.rb

```diff
@@ -0,0 +1,54 @@
+require 'rake/testtask'
+require 'rake/file_list'
+require 'ttnt/testtask'
+require 'ttnt/test_selector'
+require 'rugged'
+
+# Reference: https://github.com/bundler/bundler/blob/master/lib/bundler/gem_helper.rb
+module TTNT
+  class TaskHelper
+    include Rake::DSL if defined? Rake::DSL
+
+    def self.install_tasks
```

Comment:
I thought `Bundler::GemHelper#install_tasks` meant "install gem related tasks into Rake", so I also used the name. But it's possible it means "define rake tasks related to installing gems".

In latter case, you naming is clear, but I think `TTNT::TaskHelper#test_tasks` might give an impression that this method returns task array than define tasks. So, as another solution I changed the method name as `TTNT::TaskHelper#define_tasks`. Is this also clear to you?

(change is here: https://github.com/Genki-S/ttnt/commit/fb21b79a0527698214cc3ea9b4c13b187f6584f8)